validator/keymanager/v2/derived/derived.go

```diff
@@ -310,6 +313,15 @@ func (dr *Keymanager) CreateAccount(ctx context.Context, logAccountInfo bool) (s
 			"validatingKeyPath":   path.Join(dr.wallet.AccountsDir(), validatingKeyPath),
 		}).Info("Successfully created new validator account")
 	}
+
+	dr.lock.Lock()
+	defer dr.lock.Unlock()
+	// Append the new acount keyes to the account keys caches
```

Comment:
```suggestion
	// Append the new account keys to the account keys caches.
```

Reply:
Sorry for the spelling errors, i need a spellchecker in my development UI :)